src/main/java/Nick/Nick.java

```diff
@@ -0,0 +1,53 @@
+package Nick;
```

Comment:
should be in lower case

Reply:
Thanks for the catch Prof Akshay! Have made the corresponding improvements to my code.